services/api/Tweek.ApiService/Controllers/KeysController.cs

```diff
@@ -85,7 +85,7 @@ public async Task<ActionResult> GetAsync([FromRoute] string path)
 
             var values = await _tweek.GetContextAndCalculate(query, identities, _contextDriver, contextProps);
 
-            var errors = values.Where(x => x.Value.Exception != null).ToDictionary(x => x.Key, x => x.Value.Exception.Message);
+            var errors = values.Where(x => x.Value.Exception != null).ToDictionary(x => x.Key.ToString(), x => x.Value.Exception.Message);
```

Comment:
Are we sure that there isn't an additional place we use ConfigurationPath?

Reply:
Also in line 98, but it's already being `ToString`'d, couldn't find other places